packages/patternfly-4/verifyCoreVersions.js

```diff
@@ -8,7 +8,7 @@ const versions = new Map();
 const corePackageName = '@patternfly/patternfly-next';
 
 packages
-  .filter(({ location }) => location.match(/patternfly-4/))
+  .filter(({ location }) => location.match(/patternfly-4/) || location.match('/react-icons'))
```

Comment:
Do you think we need this filter at all? shouldn't all @patternfly/patternfly-next versions in this repo be the same? @dmiller9911 @priley86

Reply:
Yeah, I think they should all be at the same version. That will keep the dependencies consistent. This could potentially hit the same issue we hit with tokens. Not sure why I filtered in the first place.

Reply:
all right, i will remove the filter then